### PR TITLE
Improved user lists

### PR DESCRIPTION
**Pull request by gentlecat**

### CODE REVIEW

**Comment by gentlecat** [comment]
*http://metabrainz.mbsandbox.org/supporters*

File: metabrainz/templates/users/supporters-list.html [hunk 1]
```diff
@@ -24,7 +24,7 @@ <h1 class="page-title">Supporters</h1>
         <div class="users-grid">
           {% for user in featured_users %}
             <div class="user">
-              <a href="{{ url_for('users.tier', tier_id=tier.id) }}">
+              {% if user.website_url %}<a href="{{ user.website_url }}">{% endif %}
```
Comment (gentlecat) [inline metabrainz/templates/users/supporters-list.html]: We are currently not showing there which users have bad standing. Not sure if we should do it there like on tier page (see my change to *metabrainz/templates/users/tier.html* in this pull request).

Comment (mayhem) [inline metabrainz/templates/users/supporters-list.html]: I think we should. I doubt many people will click through to the individual tiers page. I think I might toss Zvooq into this bin, so we can see how it looks on the site. Please add this and then feel free to merge.

**Comment by mayhem** [comment]
:+1:

**Pull request closed without merging**